technic/machines/LV/geothermal.lua

```diff
@@ -105,9 +110,11 @@ minetest.register_node("technic:geothermal_active", {
 	         "technic_geothermal_side.png", "technic_geothermal_side.png", "technic_geothermal_side.png"},
 	paramtype2 = "facedir",
 	groups = {snappy=2, choppy=2, oddly_breakable_by_hand=2,
-		technic_machine=1, technic_lv=1, not_in_creative_inventory=1},
+		  technic_machine=1, technic_lv=1, not_in_creative_inventory=1, axey=2, handy=1},
```

Comment:
Minor indent / style
```suggestion
		technic_machine=1, technic_lv=1, not_in_creative_inventory=1, axey=2, handy=1},
```